lib/gutdata/models/metadata/analytical_dashboard.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+#
+# Copyright (c) 2010-2021 GutData Corporation. All rights reserved.
```

Comment:
can we even legally say this

Reply:
😂

Reply:
Oh shit. I must have accidentally caught that one in my multi-select. I'll put it back

Reply:
I think our outlook on it was to keep all the copyright notices Just in Case™